### PR TITLE
GUI for CFSPOPCON

### PR DESCRIPTION
**Pull request by allen-adastra**
I volunteered to help Dennis Whyte's fusion design course, 22.63, which is looking at a new ARC-class concept, and I decided to start a GUI front-end for CFSPOPCON. Got some nice input from the previous 22.63's core team, so shoutout to @AudreySaltzman  + Leo in particular for helping me slap it together.

@dboyer-cfs suggested I open a pull request cause this might be relevant to your guys needs rn.

Lmk your thoughts/comments. I was imagining having 22.63 students build this out, but would be good to get contributions from the other side too.

Demo video below. Currently takes maybe 15-ish seconds to re-run after every slider move. Probably some things that can be done to speed it up.

https://github.com/cfs-energy/cfspopcon/assets/13512058/05c5d3cd-3b5d-40d5-b00a-55fc4637832f

### CODE REVIEW

**Comment by hassec** [comment]
Thanks @allen-adastra for sharing this! Those are some pretty plots! 😍 
Currently quite busy but will try to get to this sometime next week. 
Feel free to ping if you don't see a review coming in by then 😉

**Comment by djbattagl** [comment]
This looks like a great idea and I'm really excited that POPCON will be a part of 22.63.   I'm tagging @nelsonand since there are plans to use this in a Columbia course as well.

**Review by hassec — COMMENTED** [review]
Hi @allen-adastra thanks again for sharing this! 👍 

I'd like to propose that we create a top level `contrib` folder that this `panel_gui.ipynb` could be moved to. I do think that this visualization could be useful, but do not - at this time - want to make maintenance of a GUI part of the cfspopcon library. Happy to revisit this as interest and potentially also the # of people helping with maintenance of it growths.  

So by creating `contrib` we have a way to share this contributed part without making it part of the core cfspopcon library.

I envision something like: 
```
contrib/
  GUI/
    - README.md
    - panel_gui.ipynb
    BARC/
      - gui.yaml
``` 

The changes to `make_plot.py` are of course minimal enough that I don't see a problem with those. 

And as for adding more example cases, I think that's fine too as long as we can add a test for them. 

Let me know what you think 😊

File: example_cases/BARC/gui.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+sliders:
```
Comment (allen-adastra) [inline example_cases/BARC/gui.yaml:1]: @AudreySaltzman feelings on including a BARC example? Happy to delete it if ya'll don't feel ready to have one out there.

**Pull request closed without merging**